### DISCUSSION TITLE
Importing resources from an external library into a client-only plugin

### DISCUSSION THREAD

**Discussion by DamianGlowala** [nuxt/nuxt#17545] [Questions] [answered]
I have a `lottie-player.client.ts` plugin with the following import:

`import * as LottiePlayer from "@lottiefiles/lottie-player";`

It yields the following error:
![image](https://user-images.githubusercontent.com/48835293/169861128-96f86f58-3c36-4ba2-8569-fb8ea68bec16.png)

Is this expected?

**Comment by TheAlexLichter** [comment, marked as answer]
Could you add a CodeSandbox or StackBlitz?

It looks like your Nuxt plugin does not have a default `export` 🤔

**Reply by DamianGlowala** [reply]
According to their docs, Nuxt 2 should support this in the way it's been created above. There might be differences in how Nuxt 3 registers plugins and I created an issue in their repository:

https://github.com/LottieFiles/lottie-player/issues/159

**Reply by DamianGlowala** [reply]
This is how simple it was:
```
import * as LottiePlayer from '@lottiefiles/lottie-player'

export default LottiePlayer
```

I have submitted PR updating their README for Nuxt 2 and Nuxt 3 (https://github.com/LottieFiles/lottie-player/pull/160).
